src/main/java/Duke.java

```diff
@@ -1,10 +1,129 @@
+import java.util.Arrays;
+import java.util.Scanner;
 public class Duke {
+
+    public static Task[] addTask(Task[] tasks, Task task){
```

Comment:
You could rename your array of Task(s) to `TaskList` or something like that to avoid confusion. Currently you are overusing the word "task"